empyrical/stats.py

```diff
@@ -727,6 +727,39 @@ def tail_ratio(returns):
         np.abs(np.percentile(returns, 5))
 
 
+def SQN(trade_returns):
```

Comment:
other than being an abbreviation, no.
it looks like CAGR is also being defined as `cagr` in #19